src/PrestaShopBundle/Resources/views/Admin/Improve/Shipping/Preferences/preferences.html.twig

```diff
@@ -0,0 +1,42 @@
+{#**
+ * 2007-2018 PrestaShop
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2018 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+
+{% extends '@PrestaShop/Admin/layout.html.twig' %}
+
+{% set handlingForm, carrierOptionsForm = form.handling, form.carrier_options %}
+
+{% block content %}
+    <div class="container">
+        {{ form_start(form, {'attr': {'class': 'form', 'id': 'configuration_form'}}) }}
+        <div class="row">
+            {# Handling options block #}
```

Comment:
this comment is wrong, isn't?

Reply:
I don't think it's wrong - the options block is called "Handling", but it can sound differently so I'll rephrase it to avoid double meanings